development/docker-compose-example-list.rst

```diff
@@ -0,0 +1,67 @@
+Docker Compose Example List
```

Comment:
Maybe we should introduce something like `Develpment > Container > Docker` to make the whole thing a little bit more structured. In the future there might be a chapter for Kubernetes as well.

Reply:
I agree with the first and second, good idea to divide it into appropriate sections, I didn't think about it that much